src/components/Chart/LineChart/LineChart.tsx

```diff
@@ -1,53 +1,62 @@
 import { AreaChart, Area, XAxis } from "recharts";
 import { theme } from "../../../globalStyle/theme";
+import SkeletonLineChart from "../../../pages/Homepage/components/Skeleton/SkeletonLineChart";
 import { DataChart } from "../../../utils/types";
 import Chart from "../Chart";
 
 export interface LineChartProps {
   LineChartData: DataChart[];
   ChartTitle: string;
+  firstLoad: Boolean;
+  error: string;
 }
 
 const LineChart = (props: LineChartProps) => {
   return (
     <Chart>
       <Chart.Header>{props.ChartTitle}</Chart.Header>
       <Chart.Body>
-        <AreaChart width={380} height={300} data={props.LineChartData}>
-          <defs>
-            <linearGradient id="colorUv" x1="1" y1="0.1" x2="0.3" y2="1.5">
-              <stop
-                offset="5%"
-                stopColor={theme.colors.primary_blue}
-                stopOpacity={0.3}
-              />
-              <stop
-                offset="95%"
-                stopColor={theme.colors.light_blue}
-                stopOpacity={0}
-              />
-            </linearGradient>
-          </defs>
-          <Area
-            type="monotone"
-            dataKey="value"
-            stroke={theme.colors.primary_blue}
-            strokeWidth={5}
-            fillOpacity={4}
-            fill="url(#colorUv)"
-          />
-          <XAxis
-            tickLine={false}
-            axisLine={false}
-            interval="preserveStartEnd"
-            dataKey="name"
-            tick={{
-              fontSize: 13,
-              fill: `${theme.colors.purple_blue}`,
-              fontWeight: 700,
-            }}
-          />
-        </AreaChart>
+        {props.firstLoad && !props.error && !props.LineChartData.length ? (
```

Comment:
![image](https://user-images.githubusercontent.com/82322799/152010859-41fc1b0d-0897-4139-aba5-e142f7da2a3d.png)


you can do something like this , for sapreting the login of loading and actual component data